publishing-rules.yaml

```diff
@@ -3,6 +3,10 @@ skip-gomod: true
 rules:
 - destination: kubernetes
   branches:
+  - name: origin-4.3-kubernetes-1.16.2
```

Comment:
shouldn't we remove the old one at the same time?

Reply:
I wasn't sure if we want that https://github.com/openshift/origin/pull/24016#issuecomment-545872592. Will remove then.

Reply:
done